File: api/models/PromocionesModel.js
import mongoose from "mongoose";
const PromocionesSchema = new mongoose.Schema({
	code:
	{
		type: String,
		unique: true,
		required: true
	},
	descripcion: { type: String },
	expirationDate: { type: Date },
	discountPercentage:
	{
		type: Number,
		required: true
	},
	promoCodeId:{ type: String},
	couponId: {type: String },
	createdBy: 
	{
		type: mongoose.Schema.Types.ObjectId,
		ref:'usuarios',
		required: false,
	}
    
},
	{
	timestamps: true,
	strict: true,
	collection: 'promociones' });

export default mongoose.model('promociones', PromocionesSchema)

File: ventaPeliculaWeb/ventaPeliculaWeb/Models/PromocionesModel.cs
namespace ventaPeliculaWeb.Models
{
    public class PromocionesModel
    {
        public string? _id { get; set; }
        public string? code { get; set; }
        public string? descripcion { get; set; }
        public DateTime? expirationDate { get; set; }
        public float? discountPercentage { get; set; }
        public string? promoCodoId { get; set; }
        public string? couponId { get; set; }
        public UsuariosModel? createdBy { get; set; }
        public DateTime? createdAt { get; set; }
        public DateTime? updatedAt { get; set; }
    }
}
